libr/core/cmd_debug.c

```diff
@@ -3130,6 +3130,9 @@ static int cmd_debug(void *data, const char *input) {
 	case 'e':
 		r_core_debug_esil (core, input + 1);
 		break;
+	case 'g':
```

Comment:
can you add comments here ` // "dg"` so that grepping for the command is easier?